README.md

```diff
@@ -3,6 +3,54 @@
 [![Build Status](https://dev.azure.com/urbanengine/Urban%20Engine/_apis/build/status/UrbanEngineApi?branchName=master)](https://dev.azure.com/urbanengine/Urban%20Engine/_build/latest?definitionId=6&branchName=master)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+## Configuration and Deployment
+
+This section outlines information relating to configuring or deploying the UrbanEngineApi application.
+
+## Provisioning Resources in Azure
+
+For provisioning resources in Azure the following script is available if running locally:
+
+From the root directory run the following command in PowerShell:
+
+```console
+& ./scripts/UrbanEngineApi-deploy-azure.ps1
+```
+
+**NOTE:** You will need Azure PowerShell installed if you want to run this script. Documentation on this is available at [Install Azure PowerShell](https://docs.microsoft.com/en-us/powershell/azure/install-az-ps?view=azps-3.6.1).
```

Comment:
Are you using any PowerShell syntax that's only available in newer versions of PowerShell? Do we need to recommend a minimum PowerShell version to have installed?

Reply:
It's whatever is the min required documented in the Install Azure PowerShell steps mentioned in the README there at line 20. I think v5 min, personally I use PowerShell Core and am on v7 latest.

Reply:
Ah I see now, That works out great. Microsoft will recommend the version and we don't have to worry about keeping that up to date. Thank for clarifying.

Reply:
Just getting back to this. Is the heading `Provisioning Resources in Azure` meant to be a subheading under `Configuration and Deployment`? If so can you make it a 3rd level heading via `###` please? If it's mean to be separate from that then the way it is currently is fine. I just want to make sure it's clear if that's part of that section or not.